core/interfaces/status.py

```diff
@@ -662,6 +683,64 @@ def run(self, edit):
         sublime.set_timeout_async(lambda: self.launch_tool_for_file(file_paths[0]), 0)
 
 
+class GsStatusUseCommitVersionCommand(TextCommand, GitCommand):
+
+    def run(self, edit):
+        sublime.set_timeout_async(self.run_async, 0)
+
+    def run_async(self):
+        interface = ui.get_interface(self.view.id())
+        conflicts = interface.conflict_entries
+
+        sels = self.view.sel()
+        line_regions = [self.view.line(sel) for sel in sels]
+        paths = (line[4:]
+                 for reg in line_regions
+                 for line in self.view.substr(reg).split("\n") if line)
+        for path in paths:
+            if self.is_commit_version_deleted(path, conflicts):
+                self.git("rm", "--", path)
+            else:
+                self.git("checkout", "--theirs", "--", path)
+                self.stage_file(path)
+        util.view.refresh_gitsavvy(self.view)
+
+    def is_commit_version_deleted(self, path, conflicts):
+        for conflict in conflicts:
+            if conflict.path == path:
+                return conflict.working_status == "D"
+        return False
```

Comment:
Correct me if I'm wrong, but this is section is mostly copy pasted from [`interfaces/rebase.py`][1] right? Can we perhaps DRY out the code by making the "line region col index" settable?

[1]: https://github.com/divmain/GitSavvy/blob/3c5b1ab49adc9b223d0a461d3e353a2e5cbfc64b/core/interfaces/rebase.py#L851-L877

Reply:
Yes, you are right. I just copied and modified a few lines of the code. I agree we shall not repeat the code, but I am not sure the best way to refactor the path detection code.

Reply:
Let's merge this first and work on the refactoring later

Reply:
Could you please add a `# TODO: refactor this alongside interfaces.rebase. GsRebaseUseCommitVersionCommand ` comment to remind us?